### DISCUSSION TITLE
[bug]: lnd does not broadcast closing transaction when sats_per_byte is too low.

### DISCUSSION THREAD

**Discussion by AndresCas123** [lightningnetwork/lnd#8451] [Troubleshooting]
### Background

I have not been able to recover the "limbo balance" from a couple channels that I closed with force option.
The error basically is that LND shows info log with **"broadcast failed because of: mempool fee not met: -26: mempool min fee not met, 2562 < 6660"**.
The channels was moved to pendingchannels but closing transactions is not in the mempool.
Currently status channel: "chan_status_flags": "ChanStatusBorked|ChanStatusCommitBroadcasted|ChanStatusLocalCloseInitiator"

### Your environment

* version of `lnd`: lncli version 0.16.3-beta commit=v0.16.3-beta
* Operating system: Linux
* version of `bitcoind`: Bitcoin Core version v22.0.0 (release build)

### Steps to reproduce

Close a channel with --force option

### Expected behaviour

Move channels to pending channels and after some days, recover local amount in the wallet balance

### Actual behaviour
Close channels are in the pending channels and closing transactions hash is not present in the memory pool.

Tell us what happens instead

**Comment by saubyk** [comment]
Converting this to a discussion...

Advised @AndresCas123 to update LND to the latest version, increase the bitcoin node's mempool size and restart. Closing transaction has been broadcast after those actions were taken as confirmed by the user on the slack thread.
https://lightningcommunity.slack.com/archives/C6AFCN3KL/p1706709848916249
